src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -135,6 +135,8 @@ public class ConfigConstants {
 
     public static final String OPENDISTRO_SECURITY_DEFAULT_CONFIG_INDEX = ".opendistro_security";
 
+    public static final String OPENDISTRO_SECURITY_CONFIG_VERSIONS_INDEX = ".opendistro_security_config_versions";
```

Comment:
nit: should this be a function of index name? For example to fetch current index name you can check - https://github.com/opensearch-project/security/blob/55a4220e5901a85511a5f05e24064458ec67e17c/src/main/java/org/opensearch/security/dlic/rest/api/SecurityApiDependencies.java#L77-L79

``.opendistro_security`` is the default index name but it is customizable. Hence, we can either keep this index name customizable or a function of security index name